src/main/java/controller/MainController.java

```diff
@@ -0,0 +1,108 @@
+package controller;
+
+import domain.AbsenceHistory;
+import domain.Attendance;
+import domain.AttendanceState;
+import domain.Calender;
+import domain.Crew;
+import dto.AbsenceResultDto;
+import dto.AttendanceResultDto;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Map;
+import util.FileManager;
+import view.InputView;
+import view.OutputView;
+
+public class MainController {
+
+    private LocalDate today;
+    private int todayMonth;
+    private int todayDay;
+    private String todayDayOfWeek;
+    private Attendance attendance;
+
+    public void run() {
+        prepareToday();
+        String feature;
+        do {
+            feature = InputView.inputFeature(todayMonth, todayDay, todayDayOfWeek);
+            if (feature.equals("1")) {
+                attendanceCheck();
+            }
+            if (feature.equals("2")) {
+                attendanceUpdate();
+            }
+            if (feature.equals("3")) {
+                attendanceRecord();
+            }
+            if (feature.equals("4")) {
+                readAbsence();
+            }
```

Comment:
```suggestion
            if (feature.equals("1")) {
                attendanceCheck();
            } else if (feature.equals("2")) {
                attendanceUpdate();
            } else if (feature.equals("3")) {
                attendanceRecord();
            } else if (feature.equals("4")) {
                readAbsence();
            }
```
지금의 코드는 if-else와 다른 바 없고, 오히려 성능적으로 if-else 보다 좋지 않아요.
왜 우테코에서 else 예약어 사용에 제약을 걸었을까요? if-else를 사용하지 못하면 해당 분기 처리는 구현이 불가능할까요? 객체지향의 다형성에서 정답을 찾아보시면 좋겠어요~

Reply:
단순히 else 문을 사용하지 않으면 된다고 생각했는데 결론적으로 if-else와 다른게 없네요! 피드백 반영해서 리팩터링 해보겠습니다!